src/i18n/locales/es/translation.json

```diff
@@ -546,7 +546,7 @@
   },
   "components": {
     "allowance_menu": {
-      "confirm_delete": "¿Está seguro de que desea eliminar este sitio web?",
+      "confirm_delete": "¿Está seguro de que desea eliminar este sitio web? \nSi elimina este sitio web, perderá todos los registros históricos de transacciones que realiza en él, como el total de pagos y el SAT gastado.",
```

Comment:
Thanks, but no need to translate others than English. People are using [Weblate](https://hosted.weblate.org/projects/getalby-lightning-browser-extension/getalby-lightning-browser-extension/) to add missign translations and also can watch projects I think.

Reply:
Well, I saw this website during the translation test. Keep the new text in English and keep the original text in other languages，